get_ior_results.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
change permission of this file to 755.
same for mdtest parsing

Reply:
Done